.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: Elixir CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    name: Build and test
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Elixir
+      uses: erlef/setup-beam@988e02bfe678367a02564f65ca2e37726dc0268f
+      with:
+        elixir-version: '1.6'
+        otp-version: '20.0'
```

Comment:
could we use the latest elixir/otp version in this repo? 🤔

Reply:
I tried and it doesn't work out of the box. We'll need additional changes. I figured I'd get this set up first and then make a build matrix to test newer versions.